server/routes/user.js

```diff
@@ -20,6 +23,6 @@ router.post('/signin', userController.signin)
 router.post('/signup', userController.signup)
 
 /* PATCH Route to update an Profile in the Database. */
-router.patch('/updateProfile/:uid', userController.updateProfile)
+router.post('/updateProfile', userController.updateProfile)
```

Comment:
As a general rule, should it be "/update-profile" instead since routes should not be case sensitive?